doc/spec/ingredient-spec.md

```diff
@@ -1,41 +1,43 @@
-# Conventions
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
-"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
-interpreted as described in [RFC 2119][].
-
-# Roux Ingredients
-A component in the Roux ecosystem is called an *ingredient*. A Roux ingredient
-is a filesystem subtree containing an `ingredient.md` file at its root.
-Ingredients must be distributed in a *pantry*: a filesystem subtree containing
-one or more Roux ingredients.
-
-An ingredient may be small (e.g. a single button) or large (the entire coupon
-ingredient). An ingredient may include other ingredients by composition (the
-coupon uses the button ingredient, e.g.), but must not contain other
-ingredients: if an `ingredient.md` file appears in a subdirectory of an
-ingredient, it will be ignored. Roux ingredient tools must ignore
-`ingredient.md` files that appear in a subdirectory of an ingredient.
+# Roux Ingredient Spec
+
+A component in the Roux ecosystem is called an **ingredient**. An ingredient is
+a reusable, modular, and standalone component of any size.
+
+An ingredient is a filesystem subtree containing an `ingredient.md` file at its
+root. Ingredients must be distributed in a **pantry**: a filesystem subtree
+containing one or more ingredients.
+
+An ingredient can be of any size. An ingredient may *include* other ingredients
+by composition (IngredientA references IngredientB, e.g.), but must not
+*contain* other ingredients: if an `ingredient.md` file appears in a
+subdirectory of an ingredient, it will be ignored. Roux ingredient tools must
+ignore `ingredient.md` files that appear in a subdirectory of an ingredient.
 
 ## `ingredient.md`
+
 An ingredient is defined by the presence of a file named `ingredient.md` in a
 directory. The file should document the usage of the ingredient. Roux ingredient
 tools may use the first paragraph of the file as a description of the
 ingredient.
 
-## Roux ingredient path
+## Ingredient Path
```

Comment:
I believe @lawnsea's original intent here was describing the variations (internal/external pantry, with/without subfolders). Personally I prefer verbosity/showing examples as I think it makes it easier to understand all the variations you're allowed to do. I do think this could be revised, but I'd prefer to keep the list of variations.